app/harvesters/loc_harvester.rb

```diff
@@ -0,0 +1,201 @@
+##
+# Harvester for Library of Congress
+# @see krikri::Harvester
+class LocHarvester < Krikri::Harvesters::ApiHarvester
```

Comment:
Not sure it's a huge issue, but the async harvesters written by HM don't subclass `Krikri::Harvesters::ApiHarvester`.